src/gt4py/next/program_processors/runners/dace/gtir_to_sdfg.py

```diff
@@ -482,16 +554,15 @@ def add_nested_sdfg(
         # are expected to be transient data. The caller is responsible for turning
         # them into global data and map it to arrays outside the nested SDFG.
         lambda_outputs = {
-            output.dc_node.data
+            data
```

Comment:
```suggestion
            data_name
```
I would suggest to append `_name` to clearly transmit the fact that it is only the name and not the data descriptor.
I use this convention inside the optimization.
I think that for the refactoring we should sit together and discuss some uniform naming convention or at least some guide lines how to name things.